server/routes/tags.js

```diff
@@ -29,7 +29,9 @@ const createTagRouter = ({ hubTagsService }) => {
 
       data.secondaryTags = data.id;
 
-      const pageType = data.contentType === 'tags' ? 'tags' : 'tagsSeries';
+      const pageType = ['tags', 'series'].includes(data.contentType)
+        ? 'tags'
+        : 'tagsCategories';
```

Comment:
I thought we only supported secondary tags and series on this route? Do we need `tagsCategories`?

Reply:
In the interests of maintaining existing functionality, we should leave it. Raise a ticket to remove, that way any changes in functionality is coming from product.